h2o-probes.d

```diff
@@ -22,6 +22,17 @@
 
 struct st_h2o_tunnel_t;
 
+/* @appdata
+{
+    "receive_request_header": ["value"],
+    "send_response_header": ["value"],
```

Comment:
What about header field names? I think we need to discuss.

Reply:
Given that the method itself is in the `value` field, I don't think there's a privacy issue with letting the request header name be logged, but I also don't see the point in recording the `name`. I'd be fine with also tagging `name` here.

Reply:
:ok: It makes sense. We can see them with the `-a` option in debugging, anyway.